internal/magic/text.go

```diff
@@ -297,3 +299,47 @@ func HAR(raw []byte, limit uint32) bool {
 func Svg(raw []byte, limit uint32) bool {
 	return bytes.Contains(raw, []byte("<svg"))
 }
+
+// Srt matches a SubRip file.
+func Srt(in []byte, _ uint32) bool {
+	s := bufio.NewScanner(bytes.NewReader(in))
+	if !s.Scan() {
+		return false
+	}
+	// First line must be 1.
+	if s.Text() != "1" {
+		return false
+	}
+
+	if !s.Scan() {
+		return false
+	}
+	secondLine := s.Text()
```

Comment:
```suggestion
	secondLine := s.Text()
	// Timestamp format (e.g: 00:02:16,612 --> 00:02:19,376) limits secondLine
	// length to exactly 29 characters.
	if len(secondLine) != 29 {
		return false
	}
```

This change prevents `strings.Contains` and `strings.ReplaceAll` from unnecessary running and allocating memory. Example of such input:
```
1
insanely long line here, unrelated to the srt format...
```